### ISSUE TITLE
Improve docs, Add an example project that uses this

### ISSUE THREAD

**Issue by sreich** [qoomon/maven-git-versioning-extension#213]
I am finding the documentation to be quite dauntingly complex. I think it would be helpful to have a quick maven project that shows this being used in a real scenario

Would be nice to see some cleanup of the readme too. Right now it is more of a mind dumb of every possible thing you can set, versus a "quick setup" and "detailed" listing.

**Comment by qoomon** [comment]
Would be nice if you can create a PR with some suggestions for improvements

**Comment by sreich** [comment]
Of course. First I'll have to figure out how to get it working for my case at hand and I'll see what I can do

**Comment by qoomon** [comment]
Just drop me a message If I can help somehow.

**Comment by qoomon** [comment]
@sreich do you need any help?

**Comment by dermoritz** [comment]
i am a bit on the same page as @sreich the example configuration is a minimal setup that does something different than the old normal maven versioning.
An example that reproduces the old way used with maven release plugin or maven git flow plugin would be nice.
The example should also include setting/ changing some values. And why the "docs/example-configuration.xml" looks completely different. Do we need "ref" and "refs" anymore?

**Comment by qoomon** [comment]
@dermoritz hi, could you explain me what behaviour you'd like to achieve? Then I can create an example project for this.

**Comment by qoomon** [comment]
regarding the `example-configuration.xml` i totally forgot about the existence of this file, it is not up-to-date anymore, I'll remove it.

**Comment by sreich** [comment]
I'm disconnecting from this issue (feel free to repurpose or close it as you like), but I'm switching our systems from Maven to Gradle for a variety of other reasons, so no longer am in the Maven ecosystem 

Cheers

**Comment by dermoritz** [comment]
@qoomon you already helped me much in the other ticket (https://github.com/qoomon/maven-git-versioning-extension/issues/229)

in general i want to reproduce the normal maven way. My current problem is https://github.com/qoomon/maven-git-versioning-extension/issues/245

and in general handling of the configuration - how to set and change properties. E.G. it is described how what parameter change what - but there is no example where and how to use - with example in config.

**Comment by qoomon** [comment]
@sreich feel free to try https://github.com/qoomon/gradle-git-versioning-plugin
